1237-registrator_worklist/creator_adds_a_new_message.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Creator adds a new message
+
+    @1249
+    Scenario: Creator adds a new message on a DOI request
+        Given that a Creator is logged in
+        And they open My Messages
+        And they expand a DOI request
+        And they see a list of messages between Creator and Curator(s)
+        When they enter a new message
+        And they click the Send Answer button
+        Then they see that the new message is added to the list of messages
+
+    @1746
+    Scenario: Creator adds a new message on a Support request
+        Given that a Creator is logged in
+        And they open My Messages
+        And they expand a Support request
+        And they see a list of messages between Creator and Curator(s)
+        When they enter a new message
+        And they click the Send Answer button
+        Then they see that the new message is added to the list of messages
```

Comment:
veldig fint om vi kan bruke samme språk og uttrykk på lignende features.

Eksempel: https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/1223-curator_worklist/1243-curator_writes_a_message_when_declining_a_doi_request.feature

Reply:
det er jo brukt samme som https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/1237-registrator_worklist/1249-creator_adds_a_new_message_on_a_doi_request.feature, bare byttet "DOI" med "Support"..? 

kan endre litt på Givens for å bli litt mer lik det eksempelet

Reply:
eller mener du vi skal dele opp med ett scenario som bare åpner meldingsboks og en annen som sender melding? I så fall blir det 4 scenario her da, hvor eneste foskjell blir "DOI" og "Support", som høres litt overkill ut

Reply:
mente mer uttrykk ja. som f.eks: `DOI request item` istedenfor `a DOI request`, og `Support request item` istedenfor `a Support request`